Source/CesiumRuntime/Public/CesiumMetadataValueType.h

```diff
@@ -20,6 +20,11 @@ enum class ECesiumMetadataBlueprintType : uint8 {
   Array
 };
 
+// True types are cast, reintepreted, or parsed before being packed into gpu
+// types when encoding into a texture.
```

Comment:
I think this isn't the right comment style to make it show up as help text in Blueprints. It could probably be a bit more user-facing too.

Reply:
This struct wasn't intended to be BP-accessible, it isn't exposed anywhere, I removed the UENUM and BlueprintType specifier. There is an almost equivalent version of the enum in the CesiumEncodedMetadataComponent, that one is exposed and documented properly.